### ISSUE TITLE
Segfault when Pkcs11\Module is instanciated inside an included file.

### ISSUE THREAD

**Issue by gamringer** [gamringer/php-pkcs11#31]
This causes an error:

test.php:
```
<?php
include 'init.php';
var_dump($session->getInfo());
```

init.php:
```
$module = new Pkcs11\Module(getenv('PHP11_MODULE'));
$session = $module->openSession((int)getenv('PHP11_SLOT'), Pkcs11\CKF_RW_SESSION);
```

But this doesn't:

test.php:
```
<?php
$module = new Pkcs11\Module(getenv('PHP11_MODULE'));
include 'init.php';
var_dump($session->getInfo());
```

init.php:
```
$session = $module->openSession((int)getenv('PHP11_SLOT'), Pkcs11\CKF_RW_SESSION);
```

**Comment by gamringer** [comment]
It appears to be due to the order in which objects get destroyed. Basically when their shutdown function is called.

Adding the following triggers the error before the script ends
```php
unset($module);
unset($session);
```

While this prevents it from happening
```php
unset($session);
unset($module);
```

**Comment by gamringer** [comment]
Fixed in e7cb827a44a1fd71b18a8f18490bb0b822036150
